simulator/simulator-view/src/shared/components/main-map/main-map.component.ts

```diff
@@ -25,6 +36,16 @@ export class MainMapComponent implements AfterViewInit {
     private fireStationMarkerService: FirestationMarkerService
   ) {}
 
+  ngOnInit(): void {
+    this.isInMenuCreationMode = this.fireCreationService.isSettingNewElement;
+    this.$intensitySubscription =
+      this.fireCreationService.$isInCreationState.subscribe((isCreating) => {
```

Comment:
this.$intensitySubscription doit etre la souscription a this.fireCreationService.$intensity. Et donc changer l'intensité à chaque changement dans le service. 
Pour ce $isInCreationState tu devrais créer une AUTRE souscription, $isCreating par exemple.